rclpy/test/test_destruction.py

```diff
@@ -111,11 +111,7 @@ def test_destroy_subscription_asap():
             with sub.handle:
                 pass
 
-            with sub.handle:
-                node.destroy_subscription(sub)
-                # handle valid because it's still being used
-                with sub.handle:
-                    pass
+            node.destroy_subscription(sub)
```

Comment:
@ahcorde hmm, this test seems valid to me, no? Same elsewhere.

Reply:
Refer to this comment: https://github.com/ros2/rclpy/pull/747#discussion_r606395943